policies/pol/sample_paloalto.pol

```diff
@@ -0,0 +1,48 @@
+################
```

Comment:
Can someone make custom applications or is there a set list of things? If it is a set list should this have checking to make sure someone is not applying incorrect services?